tests/hash_table_tests.cpp

```diff
@@ -191,7 +191,7 @@ TEST(HashTable, BigBuild) {
   std::vector<uint32_t> host_src(host_src_tmp.begin(), host_src_tmp.end());
 
   StaticSimpleHasher<buf_size> hasher;
-  size_t bitmask_sz = (buf_size / 32) ? (buf_size / 32) : 1;
+  size_t bitmask_sz = buf_size / 32 + 1;
```

Comment:
not a bitmask anymore, isn't it?

Reply:
I'm not sure I understand what needs to be done, and whether it is necessary if we do not replace the old version of the hash table